DfE.FindInformationAcademiesTrusts/Pages/Shared/_Footer.cshtml

```diff
@@ -7,6 +7,9 @@
           <li class="govuk-footer__inline-list-item">
             <a asp-page="/Privacy" class="govuk-footer__link" data-testid="privacy-policy-link">Privacy</a>
           </li>
+          <li class="govuk-footer__inline-list-item">
+            <a asp-page="/Cookies" class="govuk-footer__link" data-testid="cookies-page-footer-link" asp-route-returnPath="@(string.IsNullOrWhiteSpace(Context.Request.Query["returnPath"]) ? Context.Request.Path + Context.Request.QueryString : Context.Request.Query["returnPath"])">Cookies</a>
```

Comment:
The `returnPath` calculation is used in multiple places (here and on `CookieBanner.cshtml`, can we move it to a common static method which is unit tested